README.md

```diff
@@ -318,6 +318,40 @@ Be sure to set the [OS field in the Pod spec](https://kubernetes.io/docs/concept
 when deploying Windows workloads. This field is used to authoritatively identify the pod OS for validation. 
 In OpenShift, it is used when enforcing OS-specific pod security standards.
 
+## Uninstalling the Windows Machine Config Operator
+
+Windows workloads and Windows nodes must be removed before uninstalling the WMCO.
+
+To uninstall WMCO from the Openshift cluster, you need to delete the subscription, the CSV, the RBAC resources, and 
+the namespace created by the installation process.
+
```

Comment:
Please link to the shared documentation for this rather than repeating the information https://docs.openshift.com/container-platform/4.17/operators/admin/olm-deleting-operators-from-cluster.html

Reply:
No longer relevant, so I spun the PR  in a different direction.